spec/models/spotlight/blacklight_configuration_spec.rb

```diff
@@ -450,4 +450,40 @@
       end
     end
   end
+
+  describe 'document_index_view_types=' do
```

Comment:
Can we change `document_index_view_types=` to `#document_index_view_types`? It seems like we're testing both getter + setters.